### ISSUE TITLE
/hide is not working

### ISSUE THREAD

**Issue by tr12345** [CreaturePhil/Showdown-Boilerplate#129]
The /hide and /show command in the path `chat-plugins/misc.js` is not working. On the recent updates, there have been crashes reported saying that `isMuted` is not defined. Here is what it says when trying to implement the older version:

```
CRASH: TypeError: Cannot read property 'isMuted' of undefined
    at User.user.getIdentity (/home/ubuntu/workspace/chat-plugins/misc.js:153:13)
    at Room.GlobalRoom.getRoomList.ChatRoom.rollLogFile.ChatRoom.onUpdateIdentity (/home/ubuntu/workspace/rooms.js:1602:30)
    at /home/ubuntu/workspace/users.js:522:18
    at Set.forEach (native)
    at User.updateIdentity (/home/ubuntu/workspace/users.js:521:16)
    at CommandContext.exports.commands.hideauth (/home/ubuntu/workspace/chat-plugins/misc.js:165:8)
    at CommandContext.run (/home/ubuntu/workspace/command-parser.js:285:28)
    at Object.exports.multiLinePattern.exports.parse (/home/ubuntu/workspace/command-parser.js:645:18)
    at Room.chat (/home/ubuntu/workspace/rooms.js:103:27)
    at User.chat (/home/ubuntu/workspace/users.js:1369:9)

Additional information:
  user = The Run
  room = staff
  message = /hide
```
I think that either `punishments.js` or `commands.js` would have to be looked at to find a bug fix.

**Comment by TheFenderStory** [comment]
Thanks for the report @tr12345. I'll have this fixed real soon.

**Comment by TheFenderStory** [comment]
Fixed in #130
